game-core/src/main/java/games/strategy/engine/framework/startup/ui/FileBackedGamePropertiesCache.java

```diff
@@ -48,11 +50,11 @@ public void cacheGameProperties(final GameData gameData) {
         cache.getParentFile().mkdirs();
       }
       try (OutputStream os = new FileOutputStream(cache);
-          ObjectOutputStream out = new ObjectOutputStream(os)) {
+          final ObjectOutputStream out = new ObjectOutputStream(os)) {
```

Comment:
`final` not required.

Reply:
yes, but latest version of save-actions looks to not respect the ignore for implicitly final exception.

It's going to be hard to keep this clean going forward. We do not have a style guide for this one, the ROI of pointing this out and fixing it is not there as I'm going to be ping-ponging with tooling. I'm on the fence to fix these.